### PR TITLE
Update categories.txt

### PR DESCRIPTION
**Pull request by imanmoghimiq30**
Add Persian language
اضافه کردن زبان پارسی

### CODE REVIEW

**Comment by mpimenov** [comment]
@imanmoghimiq30 Don't forget to sign the CLA if you want us to accept these changes.
https://maps.me/cla/

File: data/categories.txt [hunk 1]
```diff
@@ -3341,6 +3446,7 @@ vi:3Hiệu thuốc
 zh-Hans:药店|药剂师
 zh-Hant:成藥
 el:φαρμακείο|φαρμακοποιός|χορήγηση φαρμάκων
+داروخانه
```
Comment (Zverik) [inline data/categories.txt]: Please add `fa:` prefix here.

Comment (imanmoghimiq30) [reply data/categories.txt]: Tnx 🙋

**Comment by Zverik** [comment]
Thank you!

**Comment by mpimenov** [comment]
@imanmoghimiq30 Thanks. Note that this should only enable searching by farsi strings and naming the corresponding objects. If you are interested in translating the strings for the UI, the file to change is omim/strings.txt.

**Pull request merged**